components/com_tags/tmpl/tag/list_items.php

```diff
@@ -57,26 +57,24 @@ class="inputbox" onchange="document.adminForm.submit();"
             </div>
         <?php else : ?>
             <table class="com-tags-tag-list__category category table table-striped table-bordered table-hover">
-                <?php if ($this->params->get('show_headings')) : ?>
-                    <thead>
-                        <tr>
-                            <th scope="col" id="categorylist_header_title">
-                                <?php echo HTMLHelper::_('grid.sort', 'JGLOBAL_TITLE', 'c.core_title', $listDirn, $listOrder); ?>
+                <thead <?php echo $this->params->get('show_headings', '1') ? '' :  'class="visually-hidden"'; ?>>
```

Comment:
Not sure its that important but this code will result in `<thead >`
But the suggested code will produce `<thead>`
```suggestion
                <thead<?php echo $this->params->get('show_headings', '1') ? '' :  ' class="visually-hidden"'; ?>>
```

Reply:
thx